import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1414 @@
+    maf_data["FILTER"] = vcf_data.get("FILTER", "")
+    if is_germline_data:
+        maf_data["Mutation_Status"] = "GERMLINE"
+    else:
+        maf_data["Mutation_Status"] = DEFAULT_MUTATION_STATUS
+
+
+    # resolve counts and variant allele data
+    resolve_vcf_counts_data(vcf_data, maf_data, matched_normal_sample_id, tumor_sample_data_col)
+    resolve_vcf_variant_allele_data(vcf_data, maf_data)
+    resolve_vcf_matched_normal_allele_data(vcf_data, maf_data, matched_normal_sample_id)
+    return maf_data
+
+def is_valid_vcf_maf_record(maf_record):
+    return not ((maf_record["Reference_Allele"] == "N" or maf_record["Reference_Allele"] == "") and maf_record["Tumor_Seq_Allele2"] == "")
+
+def get_vcf_file_header(filename):
+    vcf_file_header = []
+    with open(filename, "r") as vcf_file:
+        for line in vcf_file.readlines():
+            if line.startswith("#CHROM"):
+                vcf_file_header = list(map(str.strip, line.replace("#", "").split("\t")))
+                break
+    if not vcf_file_header:
+        print("Could not extract VCF header from file: %s - Exiting..." % (filename))
+        sys.exit(2)
+    return vcf_file_header
+
+def extract_vcf_data_from_file(filename, center_name, sequence_source):
+    """ Load data from a VCF file. """
+    rejected_vars_by_varclass = 0
+    rejected_vars_by_mutstatus = 0
+    rejected_vars_by_inc_data = 0
+
+    (sample_id, tumor_sample_data_col, matched_normal_sample_id) = get_vcf_sample_and_normal_ids(filename)
+    is_germline_data = ("germline" in filename)
+    data_file = open(filename, "r")
+    data_reader = [line for line in data_file.readlines() if not line.startswith("#")] # ignore commented lines
+    maf_data = []
+    vcf_data_errors = []
+    print("Loading data from file: %s" % (filename))
+    for line in data_reader:
+        vcf_file_header = get_vcf_file_header(filename)
+        vcf_data = dict(zip(vcf_file_header, list(map(str.strip, line.split("\t")))))
+        # do some additional processing on the VCF columns "INFO", "FORMAT", the identified tumor_sample_data_col, and the NORMAL column (if NORMAL is present)
+        extract_vcf_format_info_data(vcf_data, tumor_sample_data_col, matched_normal_sample_id)
+        maf_record = create_maf_record_from_vcf(sample_id, center_name, sequence_source, vcf_data, is_germline_data, matched_normal_sample_id, tumor_sample_data_col)
+        if not is_valid_vcf_maf_record(maf_record):
+            vcf_data_errors.append(vcf_data)
+        maf_data.append(maf_record)
+    data_file.close()
+    print_data_loading_summary(filename, len(maf_data), rejected_vars_by_varclass, rejected_vars_by_mutstatus, rejected_vars_by_inc_data, vcf_data_errors)
+    return maf_data
+
+
+def extract_maf_data_from_file(filename, center_name, sequence_source):
+    rejected_vars_by_varclass = 0
+    rejected_vars_by_mutstatus = 0
+    rejected_vars_by_inc_data = 0
+    records_loaded = 0
+
+    header = get_file_header(filename)
```

Comment:
For GENIE in the past the addition of the sequenced samples header has been problematic when we've had to split the MAF into multiple chunks for importing. So now we just rely on the case list file instead of generating a case list from the sequenced samples header.